alerts/ldap_password_spray.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+# Copyright (c) 2014 Mozilla Corporation
+
+
+from lib.alerttask import AlertTask
+from mozdef_util.query_models import SearchQuery, TermMatch
+
+
+class AlertLdapPasswordSpray(AlertTask):
+    def main(self):
+        # TODO: evaluate what the threshold should be to detect a password spray
+        search_query = SearchQuery(minutes=60)
+
+        search_query.add_must([
+            TermMatch('category', 'ldap'),
+            TermMatch('tags', 'ldap'),
+            TermMatch('details.response.error', 'LDAP_INVALID_CREDENTIALS')
+        ])
+
+        self.filtersManual(search_query)
+        self.searchEventsAggregated('details.client', samplesLimit=10)
+
+        # TODO: evaluate what the threshold should be for this alert
```

Comment:
Can/should this also just be pulled from the alert config?

Reply:
fixed, per your suggestion